locales-pending/fix.ftl

```diff
@@ -30,7 +30,13 @@ fix-flow-data-broker-profiles-view-data-broker-profiles-view-info-on-sites = Vie
 fix-flow-data-broker-profiles-view-data-broker-profiles-more-dialog-trigger-label = More info
 fix-flow-data-broker-profiles-view-data-broker-profiles-more-dialog-title = About these profiles
 fix-flow-data-broker-profiles-view-data-broker-profiles-more-dialog-paragraph1 = Data brokers make up a $240 billion industry of gathering your personal info and selling it to anyone searching for you. They scrape your data from publicly available sources to create profiles that include your name(s), current and previous addresses, family member names, criminal history, and much more.
-fix-flow-data-broker-profiles-view-data-broker-profiles-more-dialog-paragraph2 = We found these profiles by searching 190 data broker sites for the name, location, and date of birth you provided. These profiles are currently live and available to anyone searching for you.
+# Variables:
+# $data_broker_sites_total_num is the total number of data broker sites available to scan.
+fix-flow-data-broker-profiles-view-data-broker-profiles-more-dialog-paragraph2 = 
```

Comment:
Just as a note: I think we can always expect more than `1` broker site here and other strings that are taking the total data broker count not using a variable string. Not sure if we are unnecessarily creating more work for localizers @flodolo?

Reply:
Yea, I kind of agree with this.. However, there are quite a few places in the existing fluent files where we have this 1/1+ conditions for this exact variable.. so

Reply:
Most locales have complex plural forms, ">1" doesn't mean that they have just one form.
https://www.unicode.org/cldr/charts/43/supplemental/language_plural_rules.html

Reply:
So you'd suggest we keep the conditionals? @flodolo